common/src/main/java/com/datatorrent/common/util/PubSubWebSocketClient.java

```diff
@@ -376,7 +385,8 @@ public void unsubscribe(String topic) throws IOException
    * @return subscribe num subscriber message.
    * @throws IOException
    */
-  public static <T> String constructSubscribeNumSubscribersMessage(String topic, PubSubMessageCodec<T> codec) throws IOException
+  public static <T> String constructSubscribeNumSubscribersMessage(String topic, PubSubMessageCodec<T> codec) throws
+      IOException
```

Comment:
That's fine.  But I think as you make the style changes, it's not hard to place the line breaks in where they should be, since you're making the changes anyway.